lib/segment/src/index/field_index/full_text_index/inverted_index.rs

```diff
@@ -253,4 +275,215 @@ impl InvertedIndex {
                 }),
         )
     }
+
+    pub fn build_index(
```

Comment:
Index loading. Iterate over rocksdb and build inverted index. Because immutable state does not support points addition, I load data as mutable index and then convert to immutable when needed